concent_api/core/views.py

```diff
@@ -17,6 +21,7 @@
 @api_view
 @require_POST
 def send(_request, client_message):
+    client_public_key = decode_client_public_key(_request)
```

Comment:
This change belongs in the next commit.

Reply:
How this can be moved without resetting changes of all following commits ?

Reply:
Do interactive rebase, mark the commit as `edit`, fix it and fix conflicts with other commits.